Source/WebCore/page/ElementTargetingController.cpp

```diff
@@ -906,7 +906,7 @@ Vector<ElementIdentifier> ElementTargetingController::topologicallySortElements(
     HashSet<ElementIdentifier> unprocessedIDs;
 
     const auto elementIDs = elementIDToOccludedElementIDs.keys();
-    unprocessedIDs.add(elementIDs.begin(), elementIDs.end());
+    unprocessedIDs.addAll(elementIDs);
```

Comment:
No need for the local variable any more.
```suggestion
    unprocessedIDs.addAll(elementIDToOccludedElementIDs.keys());
```